css/properties/display.json

```diff
@@ -525,46 +702,45 @@
         },
         "subgrid": {
```

Comment:
`display: subgrid` was removed in https://github.com/w3c/csswg-drafts/commit/5a98e20279e49ae61680addc03726b0a421e7df8 in favor of per-axis subgrids.

Reply:
Done in 10c92d64ca3c3eaab3287ea1ca6de97527048aed